langgraph/src/pregel/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Runnable, RunnableConfig } from "@langchain/core/runnables";
+
+export interface PregelTaskDescription {
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  input: any;
+}
+
+export interface PregelExecutableTask {
+  name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  input: any;
+  proc: Runnable;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  writes: Array<[string, any]>; // TODO: Array type may need to be changed
+  config: RunnableConfig | undefined;
+}
+
+export interface StateSnapshot {
+  /**
+   * Current values of channels
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  values: Record<string, any> | any;
+  /**
+   * Nodes to execute in the next step, if any
+   */
+  next: [string];
```

Comment:
Double check: This needs to be `Array<string>`.

Reply:
yes current type is wrong